firmware/lucidgloves-firmware/AdvancedConfig.h

```diff
@@ -1,16 +1,23 @@
 //Advanced settings, only for the pros XD
 
-#define LOOP_TIME 4 //How much time between data sends (ms), set to 0 for a good time :)
+#define LOOP_TIME 1 //How much time between data sends (ms), set to 0 for a good time :)
 #define CALIBRATION_LOOPS -1//How many loops should be calibrated. Set to -1 to always be calibrated.
 
-//Comm defines, no touchy
-#define COMM_SERIAL 0   
-#define COMM_BTSERIAL 1 
-
 //Encoding
-#define ENCODING 1
-#define ENCODE_LEGACY 0
-#define ENCODE_ALPHA  1
+#define ENCODING ENCODE_ALPHA
+
+
+//Bluetooth and WIFI advanced settings
+#define ECHO false//Should the bluetooth or wifi data be echoed over serial for debugging
+
+//ESP32 Specific Settings
+#define ESP32_DUAL_CORE true //Should inputs be handled on a separate core for communication for more accurate rotation counting
+
+#define INTERMEDIATE_CALIBRATION true //should intermediate values (used in some forms of mixing such as MIXING_SINCOS be calibrated
+//Intermediate values: If not autocalibrating, you may need to adjust these for accuracy
+#define INTER_MAX 1500 //only used if intermediate calibration is false
+#define INTER_MIN 1200 //only used if intermediate calibration is false
```

Comment:
It seems something might be wrong with your branch? I'm not sure why git is saying you've added these lines as they already seem to be in the file.